### PR TITLE
Fix some testcases

### PR DESCRIPTION
**Pull request by fanwgwg**
- Fix a testcase that fails because function `convertToItemData` renamed to `convertToMiscSection`
- Add a testcase for function `convertSectionToItemData`

### Reviewer
@Benglin 

### FYI
@yeexinc

### CODE REVIEW

**Comment by yeexinc** [comment]
Hi @fanwgwg , there were some test cases implemented for `convertToMiscSection` and `convertToDefaultSection` in this PR: https://github.com/DynamoDS/librarie.js/pull/41

File: test/libraryUtilitiesTest.ts [hunk 1]
```diff
@@ -631,6 +634,19 @@ describe('convertToItemData function', function () {
   });
 });
 
+describe('convertSectionToItemData function', function() {
+  it('should convert layoutElement to ItemData', function() {
```
Comment (Benglin) [inline test/libraryUtilitiesTest.ts]: Shouldn't it be `LayoutElement` instead of `layoutElement`?

**Comment by Benglin** [comment]
Minor tweaks and merge conflict fix, then this will go in.

**Comment by fanwgwg** [comment]
@Benglin I've resolved the conflicts.

**Comment by Benglin** [comment]
@fanwgwg one more comment to go

**Comment by fanwgwg** [comment]
Just resolved the comment!

**Pull request merged**